tests/stress_tests/scripts/run_memcheck.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""
+This script runs memcheck tests isolated with help of gtest_parallel. It can
+upload memory measurment results to database and generate reports.
+"""
+
+import argparse
+from glob import glob
+import json
+import logging
+import os
+import subprocess
+import sys
+
+from memcheck_upload import create_memcheck_records, upload_memcheck_records, create_memcheck_report
+
+DATABASE = 'memcheck'
+COLLECTIONS = ["commit", "nightly", "weekly"]
+
+
+def run(args, log=None, verbose=True):
```

Comment:
I propose to rename `log` to `logger`

Reply:
it is not a rename. log is an argument allowing one to supply it's own logger (in case this function is used as a library).

A `logger` in main is module name and we use general logger there.

Reply:
In this case yes, but in general import of `logging` looks like `import logging as log` or `import logging as lg`, so `log` here slightly confuses. If `logger` is not proper name, we may use `custom_logger` naming